src/box/key_def.h

```diff
@@ -47,9 +47,9 @@ extern "C" {
 extern const char *sort_order_strs[];
 
 enum sort_order {
-	SORT_ORDER_ASC = 0,
-	SORT_ORDER_DESC,
-	SORT_ORDER_UNDEF,
+	SORT_ORDER_UNDEF = -1,  /* There's no strings for negative values. */
```

Comment:
Why do you need this constant (`SORT_ORDER_UNDEF`)? Let's drop ti as well.

Reply:
Discussed offile: it's secided to keep it.